client/src/components/PreviousTripsCard/PreviousTripsCard.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import "./PreviousTripsCard.css";
+import Card from "../Card/Card";
+
+// Function based React Component
+const PreviousTripsCard = props => {
+  let { link, trips } = props;
+
+  return (
+    <Card color="purple">
+      <div className="PreviousTripCard__container">{props.children[0]}</div>
+      {trips ? (
+        <div className="PreviousTripCard__trips">
+          {props.children[1]}
+          {props.children[2]}
```

Comment:
Instead of rendering each individual item in the props.children array, you can just render `{props.children}`.